Kitodo-DataManagement/src/test/java/org/kitodo/data/elasticsearch/index/type/ProjectTypeTest.java

```diff
@@ -86,16 +88,14 @@
         fifthProjectFileGroup.setPreviewImage(false);
         projectFileGroups.add(fifthProjectFileGroup);
 
-        Process firstProcess = new Process();
+        Template firstProcess = new Template();
```

Comment:
Maybe you should even rename `firstProcess` to `firstTemplate` as it is confusing.